src/main/java/net/tiagofar78/prisonescape/game/phases/Ongoing.java

```diff
@@ -1,5 +1,20 @@
 package net.tiagofar78.prisonescape.game.phases;
 
-public class Ongoing {
+public class Ongoing extends Phase {
 
+    private boolean _clockStopped;
+    
+    public Ongoing(boolean clockStopped) {
+        _clockStopped = clockStopped;
+    }
+
+    @Override
+    public Phase next() {
+        return new Finished(_clockStopped);
+    }
+
+    @Override
+    public boolean isClockStopped() {
+        return _clockStopped;
```

Comment:
Usually when using State desing pattern functions are expected to have already defined values instead of a variable. In this case it shoud return true because you want the clock running while the game is ongoing.